### PR TITLE
feat: translate test-quickstart

### PR DESCRIPTION
**Pull request by LXPWing**
add en test-quickstart.md #438

### CODE REVIEW

**Comment by seeflood** [comment]
Good English 👍

File: docs/_sidebar.md [hunk 1]
```diff
@@ -78,6 +78,7 @@
   - [FaaS design](en/design/faas/faas-poc-design.md)
 - Contributing
   - [Document Contribution Guide](en/development/contributing-doc.md)
+    - [Automate testing of Quickstart documentation with tools](en/development/test-quickstart.md)
```
Comment (seeflood) [inline docs/_sidebar.md:81]: We can make the directory structure same with [chinese doc](https://mosn.io/layotto/#/zh/development/contributing-doc) :
image

**Review by seeflood — CHANGES_REQUESTED** [review]

File: docs/_sidebar.md (hunk 1 above)
Comment (LXPWing) [reply docs/_sidebar.md]: update

File: docs/en/development/test-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+Let's use the tool to test the documentation automatically!
+
+## Principle
+
+Use the tool to execute all shell scripts in a markdown document in order.
+~~~markdown
+```shell
+```
+~~~
+Wrapped up script.
```
Comment (seeflood) [inline docs/en/development/test-quickstart.md:22]: I feel this sentence has some grammatical problems 🤔 
How about this:
~~~
Use the tool to execute all shell scripts in a markdown file sequentially, i.e. all scripts wrapped in:

```shell
```
~~~

File: docs/en/development/test-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+Note that it will not be executed with.
+~~~markdown
+```bash
+```
+~~~
+Wrapped up script.
```
Comment (seeflood) [inline docs/en/development/test-quickstart.md:29]: Same as above

**Review by seeflood — CHANGES_REQUESTED** [review]

File: docs/en/development/test-quickstart.md (hunk 2 above)
Comment (LXPWing) [reply docs/en/development/test-quickstart.md:22]: ‘i.e.’ , What does it mean?

Comment (seeflood) [reply docs/en/development/test-quickstart.md:22]: It means "that is","也就是，即"
https://dictionary.cambridge.org/zhs/%E8%AF%8D%E5%85%B8/%E8%8B%B1%E8%AF%AD/ie
https://dictionary.cambridge.org/zhs/%E8%AF%8D%E5%85%B8/%E8%8B%B1%E8%AF%AD-%E6%B1%89%E8%AF%AD-%E7%B9%81%E4%BD%93/i-e

File: docs/en/development/test-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+
+## Principle
+
+Use the tool to execute all shell scripts in a markdown file sequentially, i.e. all scripts wrapped in:
+~~~markdown
+```shell
+```
+~~~
+
+~~~markdown
```
Comment (seeflood) [inline docs/en/development/test-quickstart.md:23]:
```suggestion
Note: The script wrapped in `bash` blocks will NOT be run.
```

**Review by seeflood — CHANGES_REQUESTED** [review]

**Review by seeflood — APPROVED** [review]
lgtm

**Review by zhenjunMa — APPROVED** [review]

**Review by wenxuwan — APPROVED** [review]

**Comment by seeflood** [comment]
Thanks!

**Pull request merged**